### PR TITLE
Fabo/infinite scroll

### PR DESCRIPTION
**Pull request by faboweb**
**Description:**

Adds infinite scrolling to validator list

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/staking/TableValidators.vue [hunk 1]
```diff
@@ -178,13 +182,24 @@ export default {
       handler() {
         this.$store.dispatch(`getRewardsFromMyValidators`)
       }
+    },
+    "sort.property": function() {
+      this.showing = 20
```
Comment (jbibla) [inline src/components/staking/TableValidators.vue]: reduce this to 15?

**Comment by faboweb** [comment]
Dude! Please compare production against this.
https://12033-99653950-gh.circle-artifacts.com/0/tmp/voyager/dist/index.html#/staking/validators

**Comment by faboweb** [comment]
GraphQL will help reduce these computed lists as we get the delivered data from the backend I assume.

Removing `yourValidators` though. Good catch. 👍

**Comment by colw** [comment]
Resolved via phone :) 

It definitely is faster, for sure. Let's put it in. 👍

**Comment by colw** [comment]
And good point about the `my delegations` and back again flow. 👍

**Pull request merged**